client_cli/src/main.rs

```diff
@@ -868,8 +875,12 @@ mod json {
 
     impl RunArgs for Args {
         fn run(self, cfg: &ClientConfiguration) -> Result<()> {
-            let reader = BufReader::new(stdin());
-            let instructions: Vec<Instruction> = serde_json::from_reader(reader)?;
+            let mut reader = BufReader::new(stdin());
+            let mut raw_content = Vec::new();
+            reader.read_to_end(&mut raw_content)?;
+
+            let content = String::from_utf8(raw_content)?;
+            let instructions: Vec<Instruction> = json5::from_str(&content)?;
```

Comment:
Genesis doesn't exactly need it, but it doesn't hurt.

Reply:
I think it may be usefull for tutorials and debugging, where comments will be very helpful